components/AcademicProgramsSection/index.tsx

```diff
@@ -0,0 +1,293 @@
+                </svg>
+              </span>
+            </div>
+
+            <div className={`${styles.badge}`}>Bachelor</div>
+          </div>
+
+          <div className={`${styles.card}`}>
+            <div className={`${styles.cardTitle}`}>
+              <h2>Electronics & Instrumentation</h2>
+              <span className={` ${styles.arrowIcon}`}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  height="23"
+                  viewBox="0 96 960 960"
+                  width="23"
+                >
+                  <path d="m561 814-43-42 168-168H160v-60h526L517 375l43-42 241 241-240 240Z" />
+                </svg>
+              </span>
+            </div>
+
+            <div className={`${styles.badge}`}>Bachelor</div>
+          </div>
+
+          <div className={`${styles.card}`}>
+            <div className={`${styles.cardTitle}`}>
+              <h2>Electronics & Telecommunication</h2>
+              <span className={`${styles.arrowIcon}`}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  height="23"
+                  viewBox="0 96 960 960"
+                  width="23"
+                >
+                  <path d="m561 814-43-42 168-168H160v-60h526L517 375l43-42 241 241-240 240Z" />
+                </svg>
+              </span>
+            </div>
+
+            <div className={`${styles.badge}`}>Bachelor</div>
+          </div>
+
+          <div className={`${styles.card}`}>
+            <div className={`${styles.cardTitle}`}>
+              <h2>Information Technology</h2>
+              <span className={` ${styles.arrowIcon}`}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  height="23"
+                  viewBox="0 96 960 960"
+                  width="23"
+                >
+                  <path d="m561 814-43-42 168-168H160v-60h526L517 375l43-42 241 241-240 240Z" />
+                </svg>
+              </span>
+            </div>
+
+            <div className={`${styles.badge}`}>Bachelor</div>
+          </div>
+
+          <div className={`${styles.card}`}>
+            <div className={`${styles.cardTitle}`}>
+              <h2>Mechanical Engineering</h2>
+              <span className={`${styles.arrowIcon}`}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  height="23"
+                  viewBox="0 96 960 960"
+                  width="23"
+                >
+                  <path d="m561 814-43-42 168-168H160v-60h526L517 375l43-42 241 241-240 240Z" />
+                </svg>
+              </span>
+            </div>
+
+            <div className={`${styles.badge}`}>Bachelor</div>
+          </div>
+
+          <div className={`${styles.card}`}>
+            <div className={`${styles.cardTitle}`}>
+              <h2>Civil Engineering</h2>
+              <span className={`${styles.arrowIcon}`}>
+                <svg
```

Comment:
Don't use SVG values directly here, use `<Image />` and inside src use the SVG path
Or if you really have an utility of SVG, create a separate component that return a SVG file.